### ISSUE TITLE
[QUEST] Stinglasher

### ISSUE THREAD

**Issue by Kalasbyxan** [wow-mania/Redemption#232]
In the quest Stinglasher [7731] Stinglasher seems to be spawning underground, dont know how often but he does.

**Comment by FluoxetineWM** [comment]
Yeah I found him two days ago - he has an awful spawn and patrol path. I found him entering the left hive after about an hour of searching above ground.

**Comment by Kalasbyxan** [comment]
Hes been under the world for me 2 times now, if hes in the cave that.s np imo.
